pkg/controller/connectionsecret/connectionsecrets.go

```diff
@@ -215,6 +214,7 @@ func IsCloudGovDomain(ctx *workflow.Context) bool {
 		"cloud-qa.mongodbgov.com",
 	}
 
+	fmt.Printf("DBG: DOMAIN: %s", ctx.Client.BaseURL.Host)
```

Comment:
let's remove this

Reply:
Sure. This is just to see if we getting in there in the CI/CD